sonar-plugin-api/src/main/java/org/sonar/api/SonarQubeVersion.java

```diff
@@ -117,36 +115,17 @@
  *
  *
  * @since 5.5
+ * @deprecated since 6.0 replaced by {@link RuntimeApiVersion}
  */
 @BatchSide
 @ServerSide
 @ComputeEngineSide
 @Immutable
-public class SonarQubeVersion {
-
-  /**
-   * Constant for version 5.5
-   */
-  public static final Version V5_5 = Version.create(5, 5);
-
-  /**
-   * Constant for version 5.6
-   */
-  public static final Version V5_6 = Version.create(5, 6);
-
-  private final Version version;
-
-  public SonarQubeVersion(Version version) {
-    requireNonNull(version);
-    this.version = version;
-  }
-
-  public Version get() {
-    return this.version;
-  }
+@Deprecated
+public class SonarQubeVersion extends RuntimeApiVersion {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)